components/epaxos/src/command/command.rs

```diff
@@ -1,25 +1,30 @@
+use super::super::data;
+
 #[cfg(test)]
-#[path = "./tests/command_tests.rs"]
+#[path = "tests/command_tests.rs"]
 mod tests;
 
 // FIXME(lsl): for simplicity, string type for both key and value
 pub type Key = String;
 pub type Value = String;
 
-pub enum OpCode {
-    NoOp,
-    Set,
-    Put,
-    Get,
-}
+// re-export enum OpCode in data/command_data.rs
+pub use data::OpCode;
 
-pub struct Command {
-    pub op: OpCode,
-    pub key: Key,
-    pub value: Value, // TODO(lsl): must consider type, value and max
-}
+// re-export struct Command in data/command_data.rs
+pub use data::Command;
+
+impl Command {
+    pub fn new_command(op: OpCode, key: &str, value: &str) -> Command {
```

Comment:
一般好像直接new就行了. `Command::new()` 酱
```suggestion
    pub fn new(op: OpCode, key: &str, value: &str) -> Command {
```

Reply:
这个Command是protobuf类型，里面有个new方法。但是它的new方法没有参数，要求调用set_xxx来设置xxx field的值。

所以这里就避开了这个名字，提供了参数